<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummaryGraph</class>
 <widget class="QDialog" name="SummaryGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>492</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SummaryGraph</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Category :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cbCategory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Product :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cbProduct">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="m_tbZoomIn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="expenses.qrc">
           <normaloff>:/eXpenses/Resources/add.png</normaloff>:/eXpenses/Resources/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_tbZoomOut">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="expenses.qrc">
           <normaloff>:/eXpenses/Resources/remove.png</normaloff>:/eXpenses/Resources/remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="f4Graphics" name="m_graph">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>643</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>f4Graphics</class>
   <extends>QWidget</extends>
   <header>f4graphics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="expenses.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_cbProduct</sender>
   <signal>activated(QString)</signal>
   <receiver>SummaryGraph</receiver>
   <slot>onProductSelected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cbCategory</sender>
   <signal>activated(QString)</signal>
   <receiver>SummaryGraph</receiver>
   <slot>onCategotySelected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SummaryGraph</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onProductSelected(QString)</slot>
  <slot>onCategotySelected(QString)</slot>
 </slots>
</ui>
